segment_pointcloud_topic: "/colored_cloud_segmented"
world_frame: "world"
sensor_frame: "navigation_module_lidar_frame"

voxel_size: 0.02
voxels_per_side: 16
voxel_carving_enabled: false
allow_clear: false
anti_grazing: false #true
truncation_distance_factor: 10.0 #10.0
use_const_weight: true #false
max_ray_length_m: 30

using_ground_truth_segmentation: false

object_tracking:
  enable: false
  icp_use_reciprocal_correspondences: true
  icp_max_correspondence_distance: 1000
  icp_use_symmetric_objective: false
  icp_max_iterations: 100
  icp_transformation_epsilon: 0.000002
  icp_absolute_mse: 0.000000000001
  icp_euclidean_fitness_epsilon: 0.0001 # 0.01 = improvement of 1% of previous MSE.

meshing:
  update_mesh_every_n_sec: 5.0
  publish_mesh: true
  mesh_filename: "tpp_map.ply"

visualizer:
  enable: false
  camera_intrinsics: [462.1379497504639, 0.0, 320.5, 0.0, 462.1379497504639, 240.5, 0.0, 0.0, 1.0]
  write_frames_to_file: false
  export_path: ""

debug:
  verbose_log: false
